test/compilable/extra-files/dtoh-postscript.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+diff -pu --strip-trailing-cr ${EXTRA_FILES}/${TEST_NAME}.out ${OUTPUT_BASE}.out
+
+rm -f ${OUTPUT_BASE}.out
```

Comment:
Use `trap`

Reply:
Maybe this should be omitted entirely, see https://github.com/dlang/dmd/pull/9971#discussion_r304808436

Reply:
I think these tests should use `TEST_OUTPUT` anyway s.t. we could use #10781 to handle platform based differences (and have fewer bash tests)

Reply:
Done